andela/ah-the-answer-frontend
# #164829335 user notifications

#### What does this PR do?
Implement User notification functionality

#### Description of the task to be completed
- Enable authenticated users to view their notification when;
    - A user being followed comments on an article.
    - A user comments on an article that you have favourited
- Enable users a user to view all the notifications.
- Counts the number of the available notifications
#### How should this be manually tested
- Clone the repo and navigate to the root directory using a terminal
- Install all the dependencies by typing `npm install`
- Start the server by typing `npm start`
- Navigate to the main page using a browser at http://localhost:3000/
<img width="1440" alt="Screenshot 2019-05-16 at 19 52 50" src="https://user-images.githubusercontent.com/26567020/57872505-7d659880-7814-11e9-9641-7b093a2a5204.png">

- Sign up and log in
<img width="1440" alt="Screenshot 2019-05-16 at 19 53 01" src="https://user-images.githubusercontent.com/26567020/57872556-9b32fd80-7814-11e9-9ecb-2b65cc7cb8fc.png">

- Once on the homepage click on the bell icon to view available notifications. 
<img width="1440" alt="Screenshot 2019-05-17 at 16 31 55" src="https://user-images.githubusercontent.com/22986772/57931563-6b8dff00-78c1-11e9-90ee-57f53adb4a20.png">

- You should see the list of the available notification if any.

<img width="1440" alt="Screenshot 2019-05-17 at 16 32 07" src="https://user-images.githubusercontent.com/22986772/57931596-8496b000-78c1-11e9-87e6-d443df6488da.png">


#### Relevant PT story
https://www.pivotaltracker.com/story/show/164829335
